baus/variables.py

```diff
@@ -368,71 +368,60 @@ def vmt_nonres_cat(parcels, vmt_fee_categories):
 
 # residential fees
 @orca.column('parcels', cache=True)
-def vmt_res_fees(parcels, policy):
+def vmt_res_fees(parcels, policy, run_setup):
```

Comment:
I agree it's not the clearest implementation. I left it that way to stick with the current implementation and focus on purely creating a single-i/o structure and `run_setup`; I'm not sure if removing it as a parcels column altogether would break the model, but it could certainly become a task to change this.